lib/prng/fastrandombytes.cpp

```diff
@@ -4,44 +4,29 @@
  * Public Domain
  */
 
-#include "nfl/prng/crypto_stream_salsa20.h"
-#include "nfl/prng/randombytes.h"
 #include <inttypes.h>
 #include <iostream>
+#include "nfl/prng/crypto_stream_salsa20.h"
+#include "nfl/prng/randombytes.h"
+
+namespace nfl {
 
 static int init = 0;
-static unsigned char key[crypto_stream_salsa20_KEYBYTES];
-static unsigned char nonce[crypto_stream_salsa20_NONCEBYTES] = {0};
+static unsigned char key[32];
```

Comment:
I don't know either. giving a name is nice if the same value is used at different places and has some explicit meaning.